system/kube-monitoring/charts/prometheus-collector/aggregation.rules

```diff
@@ -4,3 +4,9 @@ aggregated:container_cpu_user_seconds_total{} = sum(container_cpu_user_seconds_t
 aggregated:swift_cluster_storage_used_percent_gauge_average{} = avg(swift_cluster_storage_used_percent_gauge) by (cluster) keep_common
 aggregated:prometheus_local_storage_ingested_samples_total_rate{} = rate(prometheus_local_storage_ingested_samples_total[5m])
 aggregated:pg_database_size_bytes_gauge_average{} = avg(pg_database_size_bytes) by (cluster,kubernetes_name,datname)
+aggregated:node_filesystem_avail{} = label_replace(node_filesystem_avail, "mountpoint", "$1", "mountpoint", "/host(/.*)")
```

Comment:
those rules replace the `/host`prefix in the label value. Altough it would be better to use the relabel_config in the prometheus config, I commented below.